### DISCUSSION TITLE
Trouble Installing on Mac

### DISCUSSION THREAD

**Discussion by astro144** [ericaltendorf/plotman#599] [Q&A]
Hi all,

I'm trying to install Plotman on my Mac, and I'm very, very green when it comes to working in Terminal.

I'm getting the following error during the Plotman installation:

"WARNING: The script plotman is installed in '/Users/...' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location."

I'm embarrassed to admit it, but I'm not even sure what it means by "not on PATH". I've tried some Googling and am in way over my head.

Any advice would be much appreciated.

**Comment by altendky** [comment]
The `PATH` is an environment variable that holds a list of directories.  When you type `chia` or other commands at a prompt the shell (the program that shows you the prompt and handles what you type) will look for a file that matches that command in each directory listed in the `PATH` environment variable.  I think in this case the warning is indicating you didn't have the virtualen environment (venv) activated when you installed.  Perhaps?  It is best if you can share the full prompt, command, and output as it helps us understand exactly what was being done when you got the message.  It probably doesn't really matter.  Just remember to activate the venv prior to running plotman.
